Automattic/wp-calypso
# Add/ List of sites in Subscription Manager (Desktop version)

Closes https://github.com/Automattic/wp-calypso/issues/74966

## Proposed Changes

As part of the Subscription Manager in Cmustinitiative, we need to implement the list of sites to the Subscription Manager. This PR adds it to the desktop version.

This is the result after these changes:

<img width="1134" alt="Screenshot 2023-03-28 at 11 45 13" src="https://user-images.githubusercontent.com/3832570/228200334-058d1ace-8ba0-4ebf-9f7b-b14ac89e7b1d.png">

## Testing Instructions

- Go to http://calypso.localhost:3000/subscriptions/sites
- Check the design matches the one in Figma: inDLaEQV8jJ21O4WXawIsJ-fi-712-15534

## Notes

- The mobile version of this list is outside the scope of this PR, and the issue for that is here: https://github.com/Automattic/wp-calypso/issues/74968
